src/main/java/com/hltech/contracts/judged/publisher/capabilities/VauntCapabilitiesReader.java

```diff
@@ -0,0 +1,86 @@
+package com.hltech.contracts.judged.publisher.capabilities;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.guava.GuavaModule;
+import com.hltech.contracts.judged.publisher.CommunicationInterface;
+import com.hltech.contracts.judged.publisher.ContractReadException;
+import com.hltech.contracts.judged.publisher.vaunt.Capabilities;
+import com.hltech.contracts.judged.publisher.vaunt.Contract;
+import com.hltech.contracts.judged.publisher.vaunt.Service;
+import lombok.extern.slf4j.Slf4j;
+
+import javax.ws.rs.core.MediaType;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.Properties;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+@Slf4j
+@CommunicationInterface("jms")
```

Comment:
I'm wondering if whole this implementation is compliant with jms or maybe specifically with activemq?

Reply:
to be discussed